### PR TITLE
[New Tracker] exitorrent.org

### PR DESCRIPTION
**Pull request by relains**

### CODE REVIEW

File: src/Jackett.Common/Definitions/exitorrent-org.yml [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+id: exitorrent-org
+name: exitorrent.org
+description: "exitorrent.org is a POLISH Private Torrent Tracker"
+language: pl-PL
+type: private
+encoding: UTF-8
+links:
+  - http://exitorrent.org/
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/exitorrent-org.yml:9]: The site supports secure connections so lets use it
```suggestion
  - https://exitorrent.org/
```

File: src/Jackett.Common/Definitions/exitorrent-org.yml [hunk 2]
```diff
@@ -0,0 +1,156 @@
+---
+id: exitorrent-org
+name: exitorrent.org
+description: "exitorrent.org is a POLISH Private Torrent Tracker"
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/exitorrent-org.yml:4]:
```suggestion
description: "exitorrent.org is a POLISH Private Torrent Tracker for MOVIES / TV / GENERAL"
```

File: src/Jackett.Common/Definitions/exitorrent-org.yml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+language: pl-PL
+type: private
+encoding: UTF-8
+links:
+  - http://exitorrent.org/
+
+caps:
+  categorymappings:
+    - {id: 1, cat: PC, desc: "Aplikacje PC"}
+    - {id: 43, cat: Books, desc: "AudioBooki"}
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/exitorrent-org.yml:14]:
```suggestion
    - {id: 43, cat: Audio/Audiobook, desc: "AudioBooki"}
```

File: src/Jackett.Common/Definitions/exitorrent-org.yml [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    - {id: 31, cat: Books, desc: "Ebooki"}
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/exitorrent-org.yml:15]:
```suggestion
    - {id: 31, cat: Books/EBook, desc: "Ebooki"}
```

File: src/Jackett.Common/Definitions/exitorrent-org.yml [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+    downloadvolumefactor:
+      case:
+        tr.darmowy: 0
+        "*": 1
+    uploadvolumefactor:
+      text: 1
+    minimumseedtime:
+      # 2 days (as seconds = 2 x 24 x 60 x 60)
+      text: 172800
+# engine n/a
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/exitorrent-org.yml:156]:
```suggestion
#TBSource v1.5.0
```

**Review by garfield69 — CHANGES_REQUESTED** [review]
is this site related to exitorrent.eu?

File: README.md [hunk 6]
```diff
@@ -329,6 +329,7 @@ A third-party Golang SDK for Jackett is available from [webtor-io/go-jackett](ht
  * Empornium (EMP) [![(invite needed)][inviteneeded]](#)
  * eShareNet
  * eStone (XiDER, BeLoad, BigTorrent)
+ * Exitorrent
```
Comment (garfield69) [inline README.md:332]:
```suggestion
 * Exitorrent.org
```
if the site is not related to exitorrent.eu then lets tell them apart
if it is related then leave as is.

**Review by garfield69 — CHANGES_REQUESTED** [review]

**Review by garfield69 — APPROVED** [review]
Fantastic. Thank you for contributing.

**Comment by garfield69** [comment]
We've missed the release windows for today, so this will go out with the next release due in about 23 hours from now.

**Comment by garfield69** [comment]
v0.21.887

**Comment by PrzemekSkw** [comment]
Hello, I try that in Jackett and test failed:
```
[exitorrent-org] (testing) Test search in exitorrent.org => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.
```
I try also in Prowlarr. Test was OK. It sync to Sonarr and Radarr and also test OK in that apps but it don't shows results. 
It shows me results only when I search in nzb360 app in Prowlarr Search engine but don't add to download client.
Open new issue for that?
Regards.

**Comment by garfield69** [comment]
@PrzemekSkw  yes, please open a new issue, and since none of the team have an account on that site, we are going to need your help to work out what is going on. 
Please turn on **enhanced** logging, repeat the test and provide the full **enhanced** log file with the new open issue, which includes the _Debug_ and _HTML_. Hopefully this will provide a clue to the problem.
See https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket for details.

**Pull request merged**